lib/testing/src/lib/process-services-cloud/resources/resources.json

```diff
@@ -262,6 +262,50 @@
       "processchild": "processchild",
       "processparent": "processparent"
     },
+    "contentModels": {
+      "contentmodel1": {
+        "name": "contentmodel1",
```

Comment:
will it be better to add some example use case name instead of cardinal numbers?

Reply:
ok, i'll try to find better names